plugins/repository-hdfs/build.gradle

```diff
@@ -229,6 +230,10 @@ if (fixtureSupported) {
   integTest.dependsOn hdfsFixture
   integTestHa.dependsOn haHdfsFixture
 
+  if (project.ext.has('inFipsJvm') && project.ext.inFipsJvm) {
```

Comment:
This condition will never be true because `project.ext.inFipsJvm` is set after this runs.
The problem here might be that we conditionally disable something else based on weather integTest is disabled or not.
In this case we need to use an `onlyIf` there as well instead of relying on a configuration time check